code-challenges.test.js

```diff
@@ -22,15 +22,22 @@ var secretCodeWord1 = "lackadaisical"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
 
-
-
-
+describe("When 'codedMssg' is called", () =>{
+  it("returns message that converts a to 4, e to 3, i to 1 and o to 0", () => {
+    expect(codedMssg(secretCodeWord1)).toEqual("l4ck4d41s1c4l")
+    expect(codedMssg(secretCodeWord2)).toEqual("g0bbl3dyg00k")
+  })
+})
 
 // b) Create the function that makes the test pass.
+// Create a function  that takes a string as an arguement
+//use replace method on string for each letter that needs to change to number.  first value in .replace is the letter to be changed with /[]/g as a global modifier (so it does not change only the first value)
 
-
-
-
+const codedMssg = (string) => {
+  return string.replace(/['a']/g,'4').replace(/['e']/g,'3').replace(/['i']/g, '1').replace(/['o']/g, '0');
```

Comment:
good chaining, is there a way to organize the data so that there's a mapping between the letters and corresponding numbers?